lib/hunter/commands/send.rb

```diff
@@ -66,25 +66,36 @@ def run
 
           request.body = data.to_json
 
-          begin
-            response = http.request(request)
-            response.value
-            puts "Successful transmission"
-          rescue Errno::ECONNREFUSED => e
-            puts "The server is unavailable"
-            puts e.message
-          rescue Net::HTTPServerException => e
-            if response.code == "401"
-              raise "Authentication key mismatch"
-            else
-              raise "Unknown HTTP error"
-            end
-          end
+          send_request(http, request)
         end
       end
 
       private
 
+      def send_request(http, request)
```

Comment:
The naming convention here is slightly misleading: the method is named `send_request`, but it is potentially sending many. I think that we should move the loop call outside of the method.